fe/fe-core/src/main/java/com/starrocks/backup/BackupJobInfo.java

```diff
@@ -241,8 +241,6 @@ public BackupTabletInfo getTablet(long tabletId) {
     public static class BackupTabletInfo {
         @SerializedName(value = "id")
         public long id;
-        @SerializedName(value = "files")
```

Comment:
are we sure we don't need this persisted? ask zichao review first

Reply:
Yes, BE downloads files using the tablet dir path. @srlch PTAL

Reply:
I have checked the restore process, it seems that only id in BackupTabletInfo will be used.